### ISSUE TITLE
[POLICY_STORE] No movement of policyId from one BPNL to other BPNL in case of Update

### ISSUE THREAD

**Issue by mkanal** [eclipse-tractusx/item-relationship-service#560]
**As** user
**I want** that policyId is not moved from one BPNL tpo other BPNL in case of update PUT 
**so that** there is a stable behavior of the api and no confusion for the user

## Hints / Details

- ...

## Acceptance Criteria

- ...

## Out of Scope

- ...

**Comment by mkanal** [comment]
Blocked by https://github.com/eclipse-tractusx/item-relationship-service/issues/561
